### ISSUE TITLE
Transceiver.java bootReceiveConnections

### ISSUE THREAD

**Issue by alan-stokes** [SpiNNakerManchester/JavaSpiNNaker#117]
_Remember: the better you describe your issue, the easier it is for us to help you._

What is the problem you see?
bootReceiveConnections is built, but never used.........
============================

What do you see that is wrong?
bootReceiveConnections is never used, building a collection for no reason

What do you think it should do instead?
use or lose

What version of our software are you using?
master
-------------------------------------------

**Comment by Christian-B** [comment]
GIGO!   Python is the same!

**Comment by alan-stokes** [comment]
then i assign rowley to fix it in spinnman. lol and donal for the java. 😛

**Comment by rowleya** [comment]
I suspect quite a lot of the transceiver is overly complicated, given that it has grown organically over time.  There was a point where we wanted to take a list of connection and use them, but I am pretty sure this functionality is never used.  Quite a lot of complexity has come from this however.  The very fact that almost every use of the transceiver uses create_transceiver_from_hostname shows this.

In summary, I am happy to leave this as a "fix later", but doubt it will be fixed any time soon (unless someone volunteers)

**Comment by alan-stokes** [comment]
chimp has volunteered. He hasnt learnt from the dave

**Comment by dkfellows** [comment]
> I suspect quite a lot of the transceiver is overly complicated

Quote of the Week!
